src/devdocs2zim/client.py

```diff
@@ -0,0 +1,189 @@
+import logging
+
+import requests
+from pydantic import BaseModel, TypeAdapter
+
+HTTP_TIMEOUT_SECONDS = 15
+
+logger = logging.getLogger(__name__)
```

Comment:
Just one side note (this is really just a remark, you can resolve the conversation without doing anything): usually we prefer to have only one logger declared in a shared module (e.g. "constants.py" or "shared.py") since this allows to easily customize log level for all scraper messages and we prefer to have only one logger name ("devdocs" here) for all scraper messages, having the exact module where the log happened is usually more an implementation detail which clutter the log and does not really help anyone (devs already know where the log is located in the source code, and non-devs have no idea what this information cam be used for). Feel free to change this in a latter PR if it makes sense to you, or do nothing if it does not.

Reply:
I pulled in constants from a future PR and added it in here with the recommended changes.

I think the approach makes sense from a UX perspective, but it is surprising from a developer perspective. My assumption was this would mostly be running in automation, but it seems like there's a nontrivial set of people who are technical enough to use the CLI, but not enough to feel very comfortable with it who might run the tool themselves. I'll make a note of that in the wiki.

Reply:
It is important to note that in addition to people not really comfortable with the CLI (which is indeed a significant share of users, at least the very vocal ones ^^), in our case the scraper will run in https://farm.openzim.org, so no need to use the CLI at all to run the scraper. And in the farm, scraper logs are shown to the farm editors, and we hope they are technical enough to read them and analyze them at a first level of interpretation should a problem occur.